tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -40,6 +40,13 @@
     private static $_queryCacheImpl = null;
 
     /**
+     * The result cache shared between all functional tests.
+     *
+     * @var \Doctrine\Common\Cache\Cache|null
+     */
+    private static $_resultCacheImpl = null;
```

Comment:
True thing, I'll keep it non-static and leave the instantiation to the test that actually uses it. Thanks